src/data/defaults.txt

```diff
@@ -120,6 +120,7 @@ global	relayAddsUseLinks	true
 global	relayAddsWikiLinks	false
 global	relayAllowRemoteAccess	false
 global	relayBrowserOnly	false
+global  relayDelayForSVN    5
```

Comment:
you've used spaces instead of tabs here.

Reply:
will fix.  how did you find that so fast

Reply:
if you view it on GitHub it looks obviously misaligned, and then I checked by highlighting the characters.